Source/CLI/Help.cpp

```diff
@@ -11,18 +11,53 @@
 //---------------------------------------------------------------------------
 
 //---------------------------------------------------------------------------
+#include "Common/Core.h"
 #include "Help.h"
 #include "Config.h"
 //---------------------------------------------------------------------------
 
 //***************************************************************************
 //
 //***************************************************************************
+//---------------------------------------------------------------------------
+String Program_Name(const String &Set)

```

Comment:
lot of `String Program = Program_Name();`, which returns default value, but it should show argv[0].